CONTRIBUTING.md

```diff
@@ -46,7 +46,8 @@ If the evidence is publicly available online, please provide the link.
 
 If the evidence is not publicly available online, make sure that you have the
 rights to share this with us, and confirm this by reading and signing the
-appropriate Contributor License Agreement (CLA):
+appropriate Contributor License Agreement (CLA); **use an email address
```

Comment:
Can you use a new sentence for this? it gets a bit hard to read. I'd do it after the bullet points as "N.B. Use an email address associated with the GitHub username you used to filed the corresponding issue" (but still in bold).